spec/factories/users.rb

```diff
@@ -28,6 +28,8 @@
     first_language_id { 1 }
     age_range_ids { [1] }
 
+    verified { false }
```

Comment:
This isn't needed if the database default is `false` yah?

Reply:
Yeah, probably not. I'll remove it.